### ISSUE TITLE
tabIndex for info. links?

### ISSUE THREAD

**Issue by vabruzzo** [vga-/wave-auto-a11y#10]
Should we have them tabbable? By default, they are not in the tab order.

**Comment by vabruzzo** [comment]
Links don't work as well for keyboard navigation. In particular, tabbing to one of these links and hitting enter does not trigger the popover. Using buttons instead. Button text cannot be copied but that doesn't seem an issue.
